### PR TITLE
guide requirements Wordpress

### PR DESCRIPTION
**Pull request by juanmaguitar**

### CODE REVIEW

**Comment by juanmaguitar** [comment]
@mburridge I've created this guide so we can use to answer questions in the Community Forum and to reduce noise in the Quick Start Guide (a link to this guide will be enough there)

Please, have a look and let me know if you have any suggestions

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
   * [Deploy Frontity on Heroku](deployment/deploy-on-heroku.md)
 * [📖 Guides](guides/README.md)
   * [Setting the URL of the WordPress data source](guides/setting-url-wordpress-source-data.md)
+  * [WordPres requirements for Frontity](guides/what-are-the-requisites-of-wordpress-for-frontity.md)
```
Comment (mburridge) [inline docs/SUMMARY.md:29]: WordPres -> WordPress

File: docs/guides/what-are-the-requisites-of-wordpress-for-frontity.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# WordPres requirements for Frontity
```
Comment (mburridge) [inline docs/guides/what-are-the-requisites-of-wordpress-for-frontity.md:1]: WordPres -> WordPress

File: docs/guides/what-are-the-requisites-of-wordpress-for-frontity.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Anyway, you can check that your WordPress REST-API is working and publicly available.
+
+The URL of this REST API may [vary depending on the type of WordPress installation](https://docs.frontity.org/guides/setting-url-wordpress-source-data) but for a typical self-hosted WordPress installation (wp.org) this URL can be got by adding `/wp-json/` to the end of your site’s URL.
+
+From that URL you should see a structure of data representing the content of your WordPress (something like [this](https://test.frontity.org/wp-json/wp/v2)). Don't worry, this is JSON and Frontity is very happy with this.
+
+### Have pretty permalinks activated
+
+Fronttiy require that the WordPress data source uses one of the pretty permalinks options, rather than the plain one, in `Settings->Permalinks`.
```
Comment (mburridge) [inline docs/guides/what-are-the-requisites-of-wordpress-for-frontity.md:27]: Fronttiy -> Frontity

**Review by mburridge — DISMISSED** [review]
Approved subject to minor typos being fixed.

**Pull request merged**